### PR TITLE
Recomputations v2

### PR DESCRIPTION
**Pull request by makistsantekidis**

### CODE REVIEW

**Comment by makistsantekidis** [comment]
please review @kaggis @dpavlos

File: app/recomputations2/recomputation_test.go [hunk 1]
```diff
@@ -0,0 +1,501 @@
...
+  "code": "200"
+ },
+ "data": {
+  "uuid": "6ac7d684-1f8e-4a02-a502-720e8f11e50b",
+  "requester_name": "John Snow",
+  "requester_email": "jsnow@wall.com",
+  "reason": "reasons",
+  "start_time": "2015-03-10T12:00:00Z",
+  "end_time": "2015-03-30T23:00:00Z",
+  "report": "EGI_Critical",
```
Comment (kaggis) [inline app/recomputations2/recomputation_test.go]: Is it possible to add multiple exclude Endpoint Groups in the examples to display the feature?

File: app/recomputations2/recomputation_test.go [hunk 2]
```diff
@@ -0,0 +1,501 @@
...
+ <status>
+  <message>Success</message>
+  <code>200</code>
+ </status>
+ <data>
+  <recomputation>
+   <uuid>6ac7d684-1f8e-4a02-a502-720e8f11e50a</uuid>
+   <requester_name>Arya Stark</requester_name>
+   <requester_email>astark@shadowguild.com</requester_email>
+   <reason>power cuts</reason>
```
Comment (kaggis) [inline app/recomputations2/recomputation_test.go]: New utility requests should support only JSON

File: .travis.yml [hunk 3]
```diff
@@ -4,16 +4,26 @@ language: go
...
+#     - mongodb-org
+#     - mongodb-org-server
+#     - mongodb-org-shell
+#     - mongodb-org-tools
+
 # install all third party go packages that we use
 install:
- - go get github.com/gorilla/handlers
- - go get github.com/gorilla/mux
+ - go get github.com/gorilla/handlers github.com/gorilla/mux
```
Comment (pkoro) [inline .travis.yml]: Maybe it's better to have this in two lines? Unless `go fmt` prefers this this way.

**Comment by pkoro** [comment]
Also please update the license in the new files added.

**Comment by kaggis** [comment]
+1

**Pull request merged**